docker/README.md

```diff
@@ -108,21 +108,46 @@ Note: please update this README with the exact wording of the error if you run i
 
 `OSError: [Errno 12] Cannot allocate memory:` could occur in conjunction with `/openlibrary/openlibrary/core` or any number of files. Simply try increasing free RAM or increasing swap/page file/virtual memory for your operating system.
 
-
 #### "No module named 'infogami'"
 
 The following should populate the target of the `infogami` symbolic link (i.e. `vendor/infogami/`):
 ```
 cd path/to/your/cloned/openlibrary
 git submodule init; git submodule sync; git submodule update
 ```
-
 Windows users may need to see [Fix line endings, symlinks, and git submodules](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#fix-line-endings-symlinks-and-git-submodules-only-for-windows-users-not-using-a-linux-vm).
 
 #### "no configuration file provided: not found" when running `docker compose <command>`
 
 Ensure you're running `docker compose` commands from within the `local-openlibrary-dev-directory`.
 
+### ConnectionError: HTTPConnectionPool(host='solr', port=8983)
+The full error is something like (line breaks added):
+```
+/openlibrary/openlibrary/templates/home/index.html: error in processing
+ template: ConnectionError: HTTPConnectionPool(host='solr', port=8983):
+Max retries exceeded with url: /solr/openlibrary/select (Caused by
+NameResolutionError("<urllib3.connection.HTTPConnection object at
+0x77a95c4e7f90>: Failed to resolve 'solr' ([Errno -2] Name or service
+not known)")) (falling back to default template)
+```
+First, verify the `solr` container is running (e.g. `docker ps | grep solr`, and then look for something like `openlibrary-solr-1` that isn't `solr-updater`.) Verify too that you can also connect to solr at http://localhost:8983/solr/#/. If you can't, try `docker compose up -d solr`, and verify that it is running and you can connect to it.
+
+If the `solr` container is running and the error persists, one cause seems to be that the containers become disconnected from `openlibrary_webnet` (though conceivably this could happen with `openlibrary_dbnet` too). `openlibrary-web-1`/`web` should be connected to both `openlibrary_webnet` and `openlibrary_dbnet`, but instead only one is connected. E.g.:
```

Comment:
An easier approach might be something like this? Do you think this would do the trick?

```sh
docker compose down

docker container ls -a
# If you see any openlibrary container here, remove them with `docker rm -f NAME`

docker compose down

docker network ls
# If you see any open library networks here, remove them with `docker network rm NAME
```

Reply:
I did try these steps, and for my part, I think doing `docker network connect openlibrary_webnet openlibrary-web-1` may be more convenient as it doesn't require deleting any containers, or even restarting them. But deleting both the containers and the networks and starting everything again with `docker compose` does indeed work.

It's also possible that my attempt to help people step through the problem and to understand what's going on made the fix appear it takes longer or would be more difficult.

If that's the case, we could remove nearly everything and just have (1) the error and )(2) something like: "Try the following: `docker compose up -d solr; docker network connect openlibrary_webnet openlibrary-web-1`, and that would require only one copy/paste to likely get back in action and no restarts or rebuilds.

There is a part of me though that wants to help people understand if we can, so either way, perhaps a compromise position could be to put a "TLDR" at the top with either the steps for stopping everything, deleting the relevant containers and networks, and starting everything back up, or (maybe) bringing up solr and reconnecting `openlibrary_webnet`, followed by more information for those who might want to understand what has potentially happened and why they're typing various things.